### ISSUE TITLE
Make command-line scripts "entry points"

### ISSUE THREAD

**Issue by goodmami** [delph-in/pydelphin#69]
The command-line scripts `pyDelphin` and `mrs.py` would be better distributed as an "entry point" defined in setup.py.

See here: http://pythonhosted.org/setuptools/setuptools.html#automatic-script-creation

See how the Xigt project does it:
- https://github.com/xigt/xigt/blob/master/setup.py#L90
- https://github.com/xigt/xigt/blob/master/xigt/main.py

**Comment by goodmami** [comment]
Example:

```
usage:
  delphin (--help | --version)
  delphin convert --from FMT --to FMT
                 [--pretty-print|--indent N]
                 [--color=(auto|always|never)]
                 [PATH]
  delphin select [--join TBL1 TBL2] DATASPEC [PATH]
  delphin mkprof [--relations PATH] [--skeleton] [PATH]
  delphin compare PROF GOLD
```

**Comment by goodmami** [comment]
This was resolved in 1ac5f52f03c4312dc51bf2d031c5878c41e260a4
